src/conf.py

```diff
@@ -49,6 +49,7 @@
 html_theme_options = {
     "footer_start": ["copyright"],
     "footer_end": ["sphinx-version"],
+    "secondary_sidebar_items": ["page-toc"],
```

Comment:
What do you think of adding back in "edit-this-page"?

Reply:
I think it didn't show up before cuz it hadn't been configured, but I poked around a bit and configured it and it's an excellent feature! Thanks for bringing attention to it